src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -73,16 +73,27 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
     }
 
     load([
-        {{#each jsDependencyPaths}}
-          '{{this}}',
-        {{/each}}
+      {{#each jsDependencyPaths}}
+        '{{this}}',
+      {{/each}}
     ], function () {
       load([
-        '{{entryBundlePath}}',
+        {{#if legacyBundlePath}}
+          '{{legacyBundlePath}}',
+        {{/if}}
         {{#each styleSheetPaths}}
           '{{this}}',
         {{/each}}
-      ]);
+      ], function () {
+        {{#unless legacyBundlePath}}
+          if (!__kbnBundles__ || !__kbnBundles__['entry/core'] || typeof __kbnBundles__['entry/core'].__kbnBootstrap__ !== 'function') {
+            console.error('entry/core bundle did not load correctly');
+            failure();
+          } else {
+            __kbnBundles__['entry/core'].__kbnBootstrap__()
+          }
+        {{/unless}}
```

Comment:
Why are we now waiting for all stylesheets to load before executing the entry point?

Reply:
Hmm, I didn't really think about that. I'll make sure that's not the case